ccd-definition/ComplexTypes/JudgeDecisionOnReconsiderationSelect-SDO-CP-nonprod.json

```diff
@@ -0,0 +1,10 @@
+[
```

Comment:
Aren't the other files called CUI-CP-nonprod?
Maybe we should follow that format?

Reply:
I have renamed the files @m-meulendijks-v1